source/renderer/app/stores/WalletsStore.js

```diff
@@ -332,6 +339,34 @@ export default class WalletsStore extends Store {
     }
   };
 
+  @action _getAccountPublicKey = async ({
+    spendingPassword: passphrase,
+  }: {
+    spendingPassword: string,
+  }) => {
+    if (!this.active || !IS_WALLET_PUBLIC_KEY_SHARING_ENABLED) {
+      return;
+    }
+
+    const walletId = this.active.id;
+    const index = '0H';
+    const extended = true;
+
+    try {
+      const walletPublicKey = await this.getAccountPublicKey.execute({
+        walletId,
+        index,
+        passphrase,
+        extended,
+      }).promise;
+      runInAction('update wallet public key', () => {
+        this.walletsPublicKeys[walletId] = walletPublicKey;
```

Comment:
@aleksandardjordjeviciohk replace with `this.activePublicKey = accountPublicKey;`

Reply:
Fixed